ibllib/pipes/base_tasks.py

```diff
@@ -277,8 +277,15 @@ def _assert_trials_data(self, trials_data=None):
             raise ValueError('No trials data and/or extractor found')
         return trials_data
 
+    def get_signatures(self, **kwargs):
+        """ explicitly adding the experiment description file to the input files for all behavior tasks
+        This is pretty hacky but avoids having to modify all the child classes"""

```

Comment:
I would rephrase the comment
```
        """ explicitly adding the experiment description file to the input files for all behavior tasks
        There is a trade-off between this inheritance design and explicit file signatures for child classes"""
 ```
 
 There are 8 child instances in ibllib alone, here both sides of the debate have a point.